### PR TITLE
Tweaks for iq

### PR DESCRIPTION
**Pull request by maurycupitt**
Quick tweaks to add host.docker.internal as the default IQ and a type in an error output

cc @bhamail / @DarthHater

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -219,7 +219,7 @@ Flags:
   -v, -- count                   Set log level, higher is more verbose
   -h, --help                     help for iq
   -a, --iq-application string    Specify public application ID for request (required)
-  -x, --iq-server-url string     Specify Nexus IQ Server URL (default "http://localhost:8070")
+  -x, --iq-server-url string     Specify Nexus IQ Server URL (default "http://host.docker.internal:8070")
```
Comment (DarthHater) [inline README.md:222]: Wildddd, does this make a pretty noticeable difference?

File: internal/customerrors/error.go [hunk 2]
```diff
@@ -45,5 +45,5 @@ func getLogFileMessage() string {
 	}
 
 	return fmt.Sprintf("For more information, check the log file at %s\n"+
-		"nancy version: %s\n", logFile, buildversion.BuildVersion)
+		"ahab version: %s\n", logFile, buildversion.BuildVersion)
```
Comment (DarthHater) [inline internal/customerrors/error.go:48]: THANK YOU FOR FIXING WHAT IS LIKELY JEFFRY BEING VERY COPY PASTA MANE (don't even have to git blame this to know my fingers and command+c +v were moving faster than my brain)

**Review by DarthHater — APPROVED** [review]
No major objections. It's just an override of a default value, not like someone can't supply something else (and using localhost:8070 is probably mostly limited to us doing local dev more than real world scenarios). Asked for some clarity on what it improves but solely to understand why, @maurycupitt and anyone in SE making this change, I'm gonna assume it makes a non trivial difference :)

File: README.md (hunk 1 above)
Comment (bhamail) [inline README.md:222]: Also curious about how this helps? (Faster, etc?)
I typically don't run IQ in a docker container, but I'm that last to claim my usage pattern is typical. ;)

Would the command `ahab config` (which allows you to set a local default value for the `iq-server-url`) be another solution?

Comment (maurycupitt) [reply README.md:222]: I don't run IQ in a docker container either - it is usually is on my team's host(laptop) - hence the default being the host.docker.internal - which says, my IQ is on my laptop, not in the container.

**Review by DarthHater — APPROVED** [review]

**Pull request merged**